raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -383,6 +390,100 @@ def handle_transferreroute(
     return TransitionResult(payment_state, events)
 
 
+def handle_transfer_reroute_light(
+    payment_state: InitiatorPaymentState,
+    state_change: ActionTransferRerouteLight,
+    channel_identifiers_to_channels: ChannelMap,
+    storage: SerializedSQLiteStorage
+) -> TransitionResult[InitiatorPaymentState]:
+
+    print("raiden/transfer/mediated_transfer/initiator_manager.py:396 running handle_transfer_reroute_light")
+
+    refund_transfer = state_change.refund_transfer
+
+    if not refund_transfer:
+        raise RaidenUnrecoverableError("Refund Transfer not found, state has changed with action "
+                                       "ActionTransferRerouteLight but refund_transfer param was not set")
+
+    store_refund_transfer = StoreMessageEvent(refund_transfer.message_identifier,
+                                              refund_transfer.payment_identifier,
+                                              1,
+                                              refund_transfer,
+                                              True,
+                                              LightClientProtocolMessageType.PaymentRefund,
+                                              refund_transfer.recipient)
```

Comment:
could you please add named parameters here for readability's sake?

Reply:
Sorry this code was replicated from somewhere else, i will do

Reply:
Done